File: 2025-redistribute-characters-to-make-all-strings-equal/redistribute-characters-to-make-all-strings-equal.py
class Solution:
    def makeEqual(self, words: List[str]) -> bool:
        store = [0] *26
        for word in words:
            for ch in word:
                store[ord(ch) -97]  +=1
        
        return all(i % len(words) == 0 for i in store)

File: 2025-redistribute-characters-to-make-all-strings-equal/redistribute-characters-to-make-all-strings-equal.cpp
class Solution {
public:
    bool makeEqual(vector<string>& words) {
        vector<int> store(26,0);

        for(string word:words){
            for(char ch:word){
                store[static_cast<int>(ch)- 97]++;
            }
        }
        for(int i=0;i<26;i++){
            if(store[i] % words.size() != 0) return false; 
        }
        return true;
    }
};